documentation/feature-flags.md

```diff
@@ -44,3 +44,9 @@ Any prerelease flags can be enabled or disabled in your agent config by adding a
 * Configuration: `{ feature_flag: { aws_bedrock_instrumentation: true|false }}`
```

Comment:
crap, i forgot to remove the docs on aws_bedrock_instrumentation. I can do that in a PR